app/src/hooks/useBioHubApi.ts

```diff
@@ -78,6 +79,23 @@ export const useBiohubApi = () => {
     return data;
   };
 
+  /**
+   * Upload project artifacts.
+   *
+   * @param projectId
+   * @param files
+   * @return {*} {Promise<IUploadProjectArtifactsResponse>}
+   */
+  const uploadProjectArtifacts = async (projectId: number, files: File[]): Promise<IUploadProjectArtifactsResponse> => {
```

Comment:
```suggestion
  const uploadProjectArtifacts = async (projectId: number, files: File[]): Promise<IUploadProjectArtifactsResponse[]> => {
```

Reply:
Actually, is this right?

Does it return an array of objects, or just an array of strings?

if it is the former, then my suggestion is correct.  But if the latter, then we dont actually need an interface, and can just say:

```typescript
const uploadProjectArtifacts = async (projectId: number, files: File[]): Promise<string[]> => {
```